namespaces/ci_job.yml

```diff
@@ -16,12 +16,12 @@ namespace:
     description: >
       Name of the CI job.
 
-  - name: build_id
+  - name: number
     type: integer
     description: >
       Build ID of the CI job.
 
-  - name: build_timestamp
+  - name: timestamp
```

Comment:
the top level `@timestamp` is the time that the job started and the `ci_job.timestamp` field is the time at which the build portion of the job started?